packages/contracts-bedrock/src/safe/DeputyPauseModule.sol

```diff
@@ -62,7 +62,7 @@ contract DeputyPauseModule is ISemver, EIP712 {
     event DeputyGuardianModuleSet(IDeputyGuardianModule indexed deputyGuardianModule);
 
     /// @notice Event emitted when the pause is triggered.
-    event PauseTriggered(address indexed deputy, bytes32 nonce);
+    event PauseTriggered(address indexed deputy, bytes32 nonce, address identifier);
```

Comment:
DPM needs to be generally updated to be installed directly into Guardian and remove references to DeputyGuardianModule